tutorials/extragalactic_gcr_photoz_catalogs.ipynb

```diff
@@ -0,0 +1,690 @@
+    "plt.subplot(121)\n",
+    "plt.scatter(brightdf['specz'],brightdf['zmode'],s=5,c='b',alpha=0.3,label=f\"all i<{magcut}\")\n",
+    "plt.scatter(brightdf['specz'][oddscut],brightdf['zmode'][oddscut],s=5,c='r',alpha=0.3,label=f\"i<{magcut} and ODDS>0.8\")\n",
+    "plt.plot([0,3],[0,3],'k--')\n",
+    "plt.xlabel(\"redshift\",fontsize=18)\n",
+    "plt.ylabel(\"photoz_mode\",fontsize=18)\n",
+    "plt.title(\"effect of photoz_odds cut\",fontsize=20)\n",
+    "plt.legend(loc='upper right',fontsize=15);\n",
+    "plt.subplot(122)\n",
+    "plt.scatter(brightdf['specz'],brightdf['zmode'],s=5,c='b',alpha=0.3,label=\"all i<23.6\")\n",
+    "plt.scatter(brightdf['specz'][chicut],brightdf['zmode'][chicut],s=5,c='r',alpha=0.3,label=\"i<23.6 and $\\chi^2$>20\")\n",
+    "plt.plot([0,3],[0,3],'k--')\n",
+    "plt.xlabel(\"redshift\",fontsize=18)\n",
+    "plt.ylabel(\"photoz_mode\",fontsize=18)\n",
+    "plt.title(\"high photoz_ml_red_chi2 cut\",fontsize=20)\n",
+    "plt.legend(loc='upper right',fontsize=15);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that a `photoz_odds` cut does, indeed remove a good deal of the outliers, but not all of them, and it also removes a good deal of the \"good\" galaxies with point estimates near the true redshift.  So, caution is warranted when using ODDS.\n",
+    "\n",
+    "Galaxies with a very high value of  `photoz_mode_ml_red_chi2` are common at redshifts ~1-1.5.  Some have reasonable redshift predictions, but we notice that many of the large outlier islands have these high chi^2 values.  Many of these galaxies will still have a high ODDS value, so a combination of ODDS and photoz_ml_red_chi2 may eliminate more outliers  However, in both cases, this is at the expense of also eliminating some good galaxies, and the potential to introduce biases in overall distributions: both ODDS and chi^2 are correlated with template/SED type, so cuts on these values can affect the distribution of the galaxies selected. Keep this in mind if you attempt to use chi^2 as a mask in your samples.\n",
+    "\n",
+    "\n",
+    "We also notice slight differences between the outlier behavior from the photoz_mode vs the photoz_mean estimate.  We will look at those degeneracies and the reasons for those differences now by examining a small number of the redshift posteriors. We'll draw from the full i<26.5 sample to show more dramatic examples that can arise with low S/N, as these better illustrate the point.  Note that switching to the i<25.0 sample would result in many more unimodal and localized PDFs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#NOTE: can't grab an @property from the linked catalog in a composite catalog!  Grab from the original catalog instead\n",
+    "xcat = GCRCatalogs.load_catalog(\"photoz_pdfs_v1_for_cosmoDC2_v1.1.4_image\")\n",
+    "zgrid = xcat.photoz_pdf_bin_centers"
```

Comment:
@sschmidt23 I realize the reason that you cannot access add-on attributes is that `only_use_master_attr` was set to True in the photo-z composite configs. In the future we should set it to False. For now we can edit it during runtime.

```suggestion
    "# NOTE: Need to set `only_use_master_attr` to False to allow accessing add-on attributes\n",
    "cat.only_use_master_attr = False\n",
    "zgrid = cat.photoz_pdf_bin_centers"
```

Reply:
done